public/data/irfanariyaz.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "Irfana Sulaiman",
+  "type": "Personal",
+  "bio": "Full-Stack developer starting to opensource",
+  "avatar": "https://github.com/irfanariyaz.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/irfanariyaz",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Linkedin",
```

I recommend capitalizing the `i` here. What do you think?
```suggestion
      "name": "Follow me on LinkedIn",
```

---

that would be great.. do I need to do a pull request after the changes are made??